allennlp/commands/serve.py

```diff
@@ -71,13 +71,18 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
                 name, description=description, help='Run the web service and demo.')
 
         subparser.add_argument('--port', type=int, default=8000)
-
+        subparser.add_argument('--archive-path', type=str, help='path to trained archive file')
```

Comment:
Instead of this, I'd recommend just changing line 36 to your archived model.  I just put the test model there to have something I could check in that would pass tests.  Do you need something that wouldn't be fixed just by changing that path?